From 462e4e5c544bcc4f30337f9004e29d9a68e62e5c Mon Sep 17 00:00:00 2001
From: "Joshua M. Clulow" <jmc@joyent.com>
Date: Mon, 30 Apr 2018 21:10:01 +0000
Subject: [PATCH] MORAY-468 "close-handles" test does not account for
 stdout/stderr Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved
 by: Cody Peter Mello <cody.mello@joyent.com>

---
 test/close-handles.test.js | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/test/close-handles.test.js b/test/close-handles.test.js
index 7cf2984..8e5f247 100644
--- a/test/close-handles.test.js
+++ b/test/close-handles.test.js
@@ -66,7 +66,9 @@ tape.test('client close actually closes sockets', function (t) {
         }
     ], function (err) {
         t.error(err);
-        t.equal(0, process._getActiveHandles().length);
+        t.deepEqual([], process._getActiveHandles().filter(function (h) {
+            return (h !== process.stderr && h !== process.stdout);
+        }));
         t.end();
     });
 });
-- 
2.21.0

